website/content/docs/troubleshooting.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Troubleshooting Guide
```

Comment:
The other one is actually also just Troubleshooting Guide. It's a good idea to distinguish the two. As I remember from our team meeting two weeks ago, you @billonahill are the expert on naming things! Suggestions?